tutorials/FITS-tables/FITS_tables.ipynb

```diff
@@ -2,17 +2,19 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "Mb0-v-tzcpN6"
+   },
    "source": [
-    "# Viewing and manipulating data from FITS tables\n",
+    "# Working with Chandra FITS tables\n",
```

Comment:
We are going to keep the word Chandra in this title, because the Chandra FITS file format is pretty unique compared to other wavelength observatories. I felt that having a more explicit title would be informative for the user.